lib/pointTransformer.js

```diff
@@ -5,6 +5,64 @@ var formatAddress = require('../lib/formatAddress');
 
 var loadDate = Date.now();
 
+var lats = [
+  'y',
+  'latitude',
+  'lat',
+  'lat_dd',
+  'lat_y'
+  ];
+
+var lons = [
+  'x',
+  'longitude',
+  'long',
+  'lon',
+  'lon_dd',
+  'long_dd',
+  'lon_x',
+  'long_x'
+];
+
+
+function getGeometry(props, geo){
+  if(geo !== null) return geo;
+
+  if(this.coordinates.x){
```

Comment:
Yep, since the geometry will need to be pulled out for every row. Turns it from O(n) to O(1)